### PR TITLE
Change sentence casing

### PR DESCRIPTION
**Pull request by bach5000**

### CODE REVIEW

File: src/CategoryEditWindow.cpp [hunk 1]
```diff
@@ -250,7 +250,7 @@ CategoryEditWindow::_OnSavePressed()
 	else
 	{
 		BAlert* alert  = new BAlert("Error",
-			"Cannot Add/Modify category. A category with the same name or color already exists.",
+			"Cannot add/modify category. A category with the same name or color already exists.",
```
Comment (humdingerb) [inline src/CategoryEditWindow.cpp]: Change to "Cannot add/modify **the** category..."

File: src/plugin/GoogleCalendar/Requests.h [hunk 2]
```diff
@@ -92,7 +92,8 @@ class Requests {
 			BMessage& responseMessage)
 		{
 			ProtocolListener listener(true);
-			BHttpRequest request(BUrl(url), true, "HTTP", &listener);
+			BUrl link(url);
+			BHttpRequest request(link,true,"HTTP",&listener);
```
Comment (humdingerb) [inline src/plugin/GoogleCalendar/Requests.h]: Style: a space after every comma.

File: src/EventWindow.cpp [hunk 3]
```diff
@@ -477,7 +477,7 @@ EventWindow::_InitInterface()
 	.End();
 	fRecurrenceBox->SetLabel("Recurrence");
 
-	fStartDateBox = new BBox("Start Date and Time");
+	fStartDateBox = new BBox("Start date and time");
```
Comment (humdingerb) [inline src/EventWindow.cpp]: This is only the internal name of the BBox. To ease scripting, better rename to "startdatetime"

File: src/EventWindow.cpp [hunk 4]
```diff
@@ -489,9 +489,9 @@ EventWindow::_InitInterface()
 			.Add(fTextStartTime, 1, 1)
 		.End()
 	.End();
-	fStartDateBox->SetLabel("Start Date and Time");
+	fStartDateBox->SetLabel("Start date and time");
 
-	fEndDateBox = new BBox("End Date and Time");
+	fEndDateBox = new BBox("End date and time");
```
Comment (humdingerb) [inline src/EventWindow.cpp]: This is only the internal name of the BBox. To ease scripting, better rename to "enddatetime"

File: src/plugin/GoogleCalendar/Requests.h (hunk 2 above)
Comment (bach5000) [reply src/plugin/GoogleCalendar/Requests.h]: Oops. I meant to do this but forgot. I'll fix these right away.

**Comment by humdingerb** [comment]
Thanks!

**Comment by bach5000** [comment]
Your welcome :) Should I try to tackle another issue?

**Comment by humdingerb** [comment]
Sure, if you find something you want to work on, go for it! Not sure if you're able to take another instance of that GCI task though...

**Pull request merged**